bitnami/mongodb/templates/hidden/statefulset.yaml

```diff
@@ -329,7 +305,7 @@ spec:
             {{- end }}
           {{- end }}
           ports:
-            - containerPort: 27017
+            - containerPort: {{ .Values.hidden.containerPorts.mongodb | int }}
               name: mongodb
```

Comment:
`extraContainerPorts` as in the other statefulsets (if you I'd like to support it)

Reply:
Thanks. It was there by mistake